### PR TITLE
[RORDEV-1027] ECK "missing authorization info" problem for ES < 8.3

### PR DESCRIPTION
**Pull request by coutoPL**
🐞Fix (ES) ["missing authorization info" problem in some situations when `xpack.security.enabled` was configured to be `true`](https://forum.readonlyrest.com/t/diana-eck/2298/75)

### CODE REVIEW

File: eck/images/rorkbn.zip
Comment (mateuszkp96) [inline eck/images/rorkbn.zip:1]: This package shouldn't be in the repository

**Review by mateuszkp96 — COMMENTED** [review]
The ROR plugin is not starting up for es80x

**Comment by coutoPL** [comment]
I did some patcher refactoring because it started to be hard to figure out what a patch does

**Review by mateuszkp96 — APPROVED** [review]
LGTM

**Pull request merged**